mdp/nodes/expansion_nodes.py

```diff
@@ -299,8 +395,12 @@ class GeneralExpansionNode(_ExpansionNode):
 
     This node has been designed to facilitate nonlinear, fixed but arbitrary  
     transformations of the data samples within MDP flows.
-        
-    **Example**::
+    
+    Original code contributed by Alberto Escalante.
+    
+    |
```

Comment:
What is this?

Reply:
Do you mean Alberto or the vertical bar? The former contributed the code, the latter results in more line space between elements rendered on the website later on. I used those whenever it would result in a nicer display.

Reply:
I meant the vertical bar ;)
Is it somehow possible to add an explaining comment?
It easily looks like a typo and could be cleaned out by a future maintainer.